integration-cli/docker_utils.go

```diff
@@ -292,14 +283,27 @@ func sockRequestRaw(method, endpoint string, data io.Reader, ct string) ([]byte,
 	var c net.Conn
 	switch daemonUrl.Scheme {
 	case "unix":
-		c, err = net.DialTimeout(daemonUrl.Scheme, daemonUrl.Path, time.Duration(10*time.Second))
+		return net.DialTimeout(daemonUrl.Scheme, daemonUrl.Path, time.Duration(10*time.Second))
```

Comment:
Am I missing something or we should be using the `timeout` parameter instead of the `time.Duration(10*time.Second) hardcoded value here? *(same issue in the `case "tcp":` below)*